src/content/docs/pt-br/basics/astro-components.mdx

```diff
@@ -138,7 +138,7 @@ const nome = 'Astro';
 <p>Espero que você tenha um dia maravilhoso!</p>
 ```
 
-Você também pode definir suas props com TypeScript, exportando uma interface de tipo `Props`. Astro vai automaticamente pegar qualquer interface `Props` exportada e dar avisos/erros de tipagem para o seu projeto. Estas props também podem dar valores padrão quando desconstruídas de `Astro.props`.
+Você também pode definir suas props com TypeScript através de uma interface de tipo `Props`. Astro vai automaticamente pegar qualquer interface `Props` definida no frontmatter e dar avisos/erros de tipagem para o seu projeto. Estas props também podem definir valores padrão quando desconstruídas de `Astro.props`.
```

Comment:
This is both a translation issue and a technical issue. You don't have to export a `Props` interface for it to be recognized by Astro; it's enough to declare it in the front-matter. Also replacing "dar" with "definir" as translation for `be given`.